Assets/EmojiTexture/TMPro/TMProEmojiAsset.cs

```diff
@@ -174,6 +174,14 @@ public static bool Process(string text)
         private static TMP_SpriteAsset CreateTMP_SpriteAsset()
         {
             var texture = new Texture2D(SHEET_SIZE, SHEET_SIZE, TextureFormat.RGBA32, false);
+
+            // Fill texture with transparent pixels
+            for (int x = 0; x < SHEET_SIZE; x++) {
+                for (int y = 0; y < SHEET_SIZE; y++) {
+                    texture.SetPixel(x, y, new Color(0, 0, 0, 0));
```

Comment:
`SetPixel` is probably slow within the loop. `SetPixels32` is probably faster, but I believe the fastest solution is to pass a byte array of zeros to `LoadRawTextureData`.
Also, all the functions above require `Apply` to upload data from CPU to GPU. Maybe the final output could look something like:
```
            var texture = new Texture2D(SHEET_SIZE, SHEET_SIZE, TextureFormat.RGBA32, false);
            texture.LoadRawTextureData(new byte[SHEET_SIZE * SHEET_SIZE * 4]);
            texture.Apply(false, EmojiTexture.CanCopyTextures);

            var spriteAsset = ScriptableObject.CreateInstance<TMP_SpriteAsset>();
            //...
```

Reply:
I'm ok if you commit your own version to master)

Reply:
Apologies for the holdup, currently working on a larger refactor and support for different platforms, so I didn't get the time to do this earlier. I will include this in the next commit :)